apps/vr-tests/src/stories/PeoplePicker.stories.tsx

```diff
@@ -96,7 +146,22 @@ storiesOf('PeoplePicker', module)
       pickerSuggestionsProps={ suggestionProps }
       disabled
     />
-  ))
+  )).add('Normal with text', () => (
+    <Screener
+      steps={ new Screener.Steps()
+        .snapshot('default', { cropTo: '.testWrapper' })
+        .click('.ms-BasePicker-input')
+        .setValue('.ms-BasePicker-input', 'a')
+        .snapshot('suggestiona')
```

Comment:
Nit: maybe name the snapshot `'suggestion: "a"'` so it's more clear what you're testing here

Normally we add `hover()` after `click()`. I forget the exact reasoning. Might be fine without it since you call `setValue()`, but when Screener finishes running, I'd check to make sure the snapshot looks as intended